interview-questions.md

```diff
@@ -7,48 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: The syntax for an instance variable is @ followed by the variables assigned name. This is (as the name reveals) is an instance of the chosen variable and not a direct call of the variable itself. 
```

Comment:
Great answer. Since the question is asking how its different from other variables, I'd want to go into some explanation of that.